build/cluster.js

```diff
@@ -180,50 +174,60 @@ gulp.task('clear-kubernetes', function () {
  * Spawns local-up-cluster.sh script.
  */
 gulp.task('spawn-cluster', ['checkout-kubernetes-version', 'kubeconfig-set-cluster-local',
-        'kubeconfig-set-context-local', 'kubeconfig-use-context-local', 'kill-cluster'],
-    function () {
-        clusterProcess = childProcess.spawn(upScript);
-
-        clusterProcess.stdout.on('data', function (data) {
-            gulpUtil.log('' + data);
-            if (!isClusterRunning) {
-                clusterHealthCheck(function (result) {
-                    if (result === 'ok') {
-                        isClusterRunning = true;
-                        gulpUtil.log(
-                            gulpUtil.colors.magenta("Kubernetes cluster is up and running."));
-                    }
-                });
-            }
-        });
+    'kubeconfig-set-context-local', 'kubeconfig-use-context-local', 'kill-cluster'],
+  function () {
+    clusterProcess = childProcess.spawn(upScript, {stdio: 'inherit'});
 
-        clusterProcess.on('exit', function () {
-            clusterProcess = null;
-        });
+    clusterProcess.on('exit', function () {
+      clusterProcess = null;
+    });
+  });
+
+
+/**
+ * Checks periodically if cluster is up and running.
+ */
+gulp.task('wait-for-cluster', function (doneFn) {
+  let isRunningHandler = setInterval(isRunning, 10000);
```

Comment:
Is this 10 seconds? Isn't it too much? Checking for healthz every second SGTM because it can speed up waiting for cluster for roughly 9 seconds, which is a lot when I run it locally :)